### ISSUE TITLE
Bug that kills luaconsole.vert shader

### ISSUE THREAD

**Issue by archiekinnane** [OpenSpace/OpenSpace#687]
I haven't been able to isolate what specifically causes this issue, but sometimes when opening OpenSpace for the first time after adding/modifying files, luaconsole.vert collapses to what I've pasted below, which causes a fatal error and crash.
```
[Debug##Default]
Pos=60,60
Size=400,400
Collapsed=0

[Scene Properties]
Pos=62,59
Size=350,500
Collapsed=0

[Space/Time]
Pos=60,60
Size=459,398
Collapsed=0
```

Then, after I replace the above with the correct luaconsole.vert code, OpenSpace loads correctly. It's happened about 5 times or so.

**Comment by hedbergj** [comment]
I've been getting the same thing recently, but it happens with different shader files, like powerScaling_vs.hglsl for example. Not sure what triggers it, but all of sudden while flying around, dozens of shader errors and the file has been overwritten with similar entries to yours.

**Comment by alexanderbock** [comment]
Thanks for the report, that was a bug with storage of temporary memory;  it is fixed in cce27a3a1e8da4ab7d8f000fa04b2d19af574bb0
